### PR TITLE
Setup general email account

### PR DESCRIPTION
**Pull request by jovirkku**
Signed-off-by: Jorma Virkkunen <jorma.virkkunen@jolla.com>

### CODE REVIEW

**Comment by jpwalden** [comment]
Looks good to me, however there is one empty space on line 128 which causes that the **bold** font doesn't work:
**Q: I can only receive emails, but cannot send any
**

**Comment by jovirkku** [comment]
> Looks good to me, however there is one empty space on line 128 which causes that the **bold** font doesn't work: **Q: I can only receive emails, but cannot send any **

Fixed in commit 816063...

**Review by jpwalden — APPROVED** [review]
Looks good to me!

File: Support/Help_Articles/Setup_General_Email_Account/README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+parent: Help Articles
+layout: default
+nav_order: 760
+---
+
+On Sailfish OS, accounts are created and managed in the menu "Settings > Accounts".
+
+By **General email** we mean all IMAP and POP3 email accounts.
+
+If the email account you wish to set up is a **Microsoft Exchange** account, please see **[this article](https://docs.sailfishos.org/Support/Help_Articles/Setup_Exchange_Account/)**. Setting up a **Google** account is explained in **[this article](https://docs.sailfishos.org/Support/Help_Articles/Setup_Google_Account/)**.
```
Comment (vigejolla) [inline Support/Help_Articles/Setup_General_Email_Account/README.md:13]: Could leave the `https://docs.sailfishos.org/` out from these links, i.e. use just `/Support/Help_Articles/Setup_Google_Account/` and `/Support/Help_Articles/Setup_Exchange_Account/`

Comment (jovirkku) [reply Support/Help_Articles/Setup_General_Email_Account/README.md]: Links shortened

**Review by vigejolla — APPROVED** [review]

**Pull request merged**